client/src/app/interfaces/transaction.ts

```diff
@@ -1,12 +1,11 @@
-import { Stock } from './stock';
-
 export interface Transaction {
   symbol: string;
-  qty: number;
-  side: string; // buy / sell
+  qty?: number;
+  total?: number;
+  side?: string; // buy / sell
   type: string; // market / limit
   time_in_force: string;
   status?: string; // This is the status of the order
   averagePricePerShare?: number;
-  stockFulfilled?: Stock;
```

Comment:
Was this the attribute that you were talking about earlier? The one that we agreed to talk to at the next meeting?

Reply:
Yes, exactly. We can bring it back but since we have status inside transaction, we can check the status and realize if it was fulfilled or not.